README.md

```diff
@@ -1 +1,62 @@
-# client-examples
\ No newline at end of file
+# Description of client-examples
+This repository contains example of Kafka clients. 
+Both producer and consumer clients are transformed into docker images which allows to deploy them on OpenShift platform.
+This may serve as basic usage example for [Strimzi][strimziGH]project.
+
+This repository also contains `deployment.yaml` file with full deployment configuration of producer, consumer, and definitions for topic and user operator.
+Logging configuration can be found in the `log4j2.properties` file for producer and consumer separately.
+
+## Build
+After cloning this repository to your folder Hello World example is fully ready to be build with maven.
+By one single command Java sources are compiled into JAR files, docker images are created and pushed to repository.
+The command is:
+    `make all`
+
+Note: Be sure `docker` and `oc` cluster where images should be pushed are running.
+
+## Usage
+This example provides Hello World use case of Strimzi Kafka Operator.
+Basic requirement to run this example is running OpenShift cluster with deployed Kafka and Zookeeper containers.
+Examples how to deploy basic configuration can be found in the [Strimzi documentation][strimziDoc].
```

Comment:
It's not just any old Kafka + Zookeeper running in OpenShift: It has to be deployed by the Strimzi Cluster Operator with the Topic Operator enabled, right?